src/components/report-filter-controls.tsx

```diff
@@ -21,15 +22,27 @@ enum Control {
   SCALE = "scale",
   AGENCY = "agency",
   BY_AGENCY = "by_agency",
+  TIME_BUCKET = "time_bucket",
```

Comment:
Another name for time bucket would be like, "period" but I wasn't sure if that was clearer or not 🤷 . Open to feedback on what to call this!